axelrod/tests/unit/test_tournament.py

```diff
@@ -745,6 +731,55 @@ def test_write_to_csv_without_results(self):
         expected_df = pd.read_csv(axl_filename(path))
         self.assertTrue(df.equals(expected_df))
 
+    @given(seed=integers(min_value=1, max_value=4294967295))
+    @settings(max_examples=5, deadline=None)
+    def test_seeding_equality(self, seed):
+        """Tests that a tournament with a given seed will return the
+        same results each time."""
```

Comment:
```suggestion
        """Tests that a tournament with a given seed will return the
        same results each time. This specifically checks when running using multiple cores so as to confirm that https://github.com/Axelrod-Python/Axelrod/issues/1277 is fixed."""
```

(Or something like that?)